pytest_rts/utils/db.py

```diff
@@ -284,3 +285,28 @@ def is_last_update_hash(self, commithash):
     def get_last_update_hash(self):
         """Return latest update hash"""
         return self.db_cursor.execute("SELECT hash FROM last_update_hash").fetchone()[0]
+
+    def get_existing_tests(self):
+        """Return all currently mapped pytest test function names"""
+        existing_tests = set()
+        for test in [
+            x[0]
+            for x in self.db_cursor.execute(
+                "SELECT context FROM test_function"
+            ).fetchall()
+        ]:
+            existing_tests.add(test)
+        return existing_tests
```

Comment:
There's no reason to create a set piece by piece in a loop.
```
return {x[0] for x in ...}
```

Reply:
Also consider this making a property:
```
@property
def existing_tests(self):
    ...
```

Reply:
Changed to property